AutomatedTesting/Gem/PythonTests/prefab/PrefabLevel_BasicWorkflow.py

```diff
@@ -36,30 +39,59 @@ def PrefabLevel_BasicWorkflow():
+
+    def print_error_if_failed(prefab_operation_result):
+        if not prefab_operation_result.IsSuccess():
+            Report.info(f'Error message: {prefab_operation_result.GetError()}')
+
+
+# Open the test level
     helper.init_idle()
-    helper.open_level("Prefab", "Base")
+    helper.open_level(TEST_LEVEL_FOLDER, TEST_LEVEL_NAME)
 
 # Create a new Entity at the root level
     new_entity_id = editor.ToolsApplicationRequestBus(bus.Broadcast, 'CreateNewEntity', EntityId())
     Report.result(Tests.create_new_entity, new_entity_id.IsValid())
 
 # Checks for prefab creation passed or not 
-    new_prefab_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'new_prefab.prefab')
-    create_prefab_result = prefab.PrefabPublicRequestBus(bus.Broadcast, 'CreatePrefabInMemory', [new_entity_id], new_prefab_file_path)
-    Report.result(Tests.create_prefab, create_prefab_result)
+    create_prefab_result = prefab.PrefabPublicRequestBus(bus.Broadcast, 'CreatePrefabInMemory', [new_entity_id], NEW_PREFAB_FILE_PATH)
+    Report.result(Tests.create_prefab, create_prefab_result.IsSuccess())
+    print_error_if_failed(create_prefab_result)
 
 # Checks for prefab instantiation passed or not 
-    container_entity_id = prefab.PrefabPublicRequestBus(bus.Broadcast, 'InstantiatePrefab', new_prefab_file_path, EntityId(), EXPECTED_NEW_PREFAB_POSITION)
-    Report.result(Tests.instantiate_prefab, container_entity_id.IsValid())
+    instantiate_prefab_result = prefab.PrefabPublicRequestBus(bus.Broadcast, 'InstantiatePrefab', NEW_PREFAB_FILE_PATH, EntityId(), EXPECTED_NEW_PREFAB_POSITION)
+    Report.result(Tests.instantiate_prefab, instantiate_prefab_result.IsSuccess() and instantiate_prefab_result.GetValue().IsValid())
+    print_error_if_failed(instantiate_prefab_result)
+    container_entity_id = instantiate_prefab_result.GetValue()
+    editor.EditorEntityAPIBus(bus.Event, 'SetName', container_entity_id, INSTANTIATED_PREFAB_NAME)
 
 # Checks if the new prefab is at the correct position and if it fails, it will provide the expected postion and the actual postion of the entity in the Editor log
     new_prefab_position = azlmbr.components.TransformBus(azlmbr.bus.Event, "GetWorldTranslation", container_entity_id)
     is_at_position = new_prefab_position.IsClose(EXPECTED_NEW_PREFAB_POSITION)
     Report.result(Tests.new_prefab_position, is_at_position)
     if not is_at_position:
         Report.info(f'Expected position: {EXPECTED_NEW_PREFAB_POSITION.ToString()}, actual position: {new_prefab_position.ToString()}')
-    
+
+# Checks for prefab deletion passed or not 
+    delete_prefab_result = prefab.PrefabPublicRequestBus(bus.Broadcast, 'DeleteEntitiesAndAllDescendantsInInstance', [container_entity_id])
+    Report.result(Tests.delete_prefab, delete_prefab_result.IsSuccess())
+    print_error_if_failed(delete_prefab_result)
+    Report.result(Tests.instantiated_prefab_removed, find_entity_by_name(INSTANTIATED_PREFAB_NAME) is None)
```

Comment:
In addition to testing that the container entity is deleted, it would be a good check to also test whether all entities in the prefab are deleted.

Reply:
Fixed.